addons/l10n_it_edi/views/report_invoice.xml

```diff
@@ -2,12 +2,19 @@
 <odoo>
     <template id="report_invoice_document" inherit_id="account.report_invoice_document">
         <div name="comment" position="before">
-            <div t-if="o.l10n_it_origin_document_type" name="pa_fields">
-                <b><span t-field="o.l10n_it_origin_document_type"/></b>: <span t-field="o.l10n_it_origin_document_name"/><br/>
-                <t t-if="o.l10n_it_origin_document_date"><b>Document Date: </b><span t-field="o.l10n_it_origin_document_date"/><br/></t>
-                <t t-if="o.l10n_it_cig"><b>CIG: </b><span t-field="o.l10n_it_cig"/><br/></t>
-                <t t-if="o.l10n_it_cup"><b>CUP: </b><span t-field="o.l10n_it_cup"/><br/></t>
-            </div>
+                <t t-if="o.l10n_it_stamp_duty"><b>Stamp Duty: </b><span t-field="o.l10n_it_stamp_duty"></span></t><br/>
+                <t t-if="o.l10n_it_ddt_id"><b>Transport Document: </b><span t-field="o.l10n_it_ddt_id"></span></t><br/>
+                <div t-if="o.l10n_it_origin_document_type" name="pa_fields">
+                    <b><span t-field="o.l10n_it_origin_document_type"/></b>: <span t-field="o.l10n_it_origin_document_name"/><br/>
+                    <t t-if="o.l10n_it_origin_document_date"><b>Document Date: </b><span t-field="o.l10n_it_origin_document_date"/><br/></t>
+                    <t t-if="o.l10n_it_cig"><b>CIG: </b><span t-field="o.l10n_it_cig"/><br/></t>
+                    <t t-if="o.l10n_it_cup"><b>CUP: </b><span t-field="o.l10n_it_cup"/><br/></t>
+                </div>
+        </div>
+
+        <div name="payment_communication" position="inside">
+            <t t-set="payments" t-value="o.line_ids.filtered(lambda line: line.account_id.account_type in ('asset_receivable', 'liability_payable'))"/>
```

Comment:
Comment 1)
```suggestion
            <t t-set="payments" t-value="o.line_ids.filtered(lambda line: line.display_type == 'payment_term')"/>
```
Comment 2)
`payments` is confusing here. At least, rename it to `term_lines`